Lesson27/Merve/update-post.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Update post</title>
+  <link rel="stylesheet" type="text/css" href="update.css" />
+
+</head>
+
+<body>
+  <!-- TODO: add form element -->
+  
+  <form id="update-post-form">
+    <label for="postId">Post ID</label>
+    <input type="text" id="postId" name="postId" required>
+    <label for="postContent">Content</label>
+    <textarea id="postContent" name="postContent" required></textarea>
+    <button type="submit">Update Post</button>
+  </form>
+  <button onclick="deletePost(1)">Delete Post</button>
```

Comment:
Remove the delete button from this page, the purpose of the page is to let the user update the post.